### DISCUSSION TITLE
ubuntu 24.04 使用疑问？

### DISCUSSION THREAD

**Discussion by wicking2233** [xiaohucode/yidaRule#252] [Q&A]
![截图 2024-12-16 22-48-40](https://github.com/user-attachments/assets/28486788-b6be-4bea-b469-312ea593722c)
使用包：
![截图 2024-12-16 22-53-13](https://github.com/user-attachments/assets/c79dc680-9342-4a54-a8f2-81def99db861)

问题：目前打开界面呈现的初始化的界面，肯定不对感觉是测试页面，不确定是包的问题，还是环境的问题，有人清楚不？

**Comment by ZenQy** [comment]
我在nixos上打包,运行也遇到了同样的问题
